### PR TITLE
Issue 2 view an entry

### PR DESCRIPTION
**Pull request by randygerada**
https://user-images.githubusercontent.com/113477527/199815683-28726829-d378-418a-9020-9d10e33297a1.mov

### CODE REVIEW

**Comment by randygerada** [comment]
Screenshot 2022-11-03 at 1 35 57 PM
Screenshot 2022-11-03 at 1 36 06 PM

File: js/main.js [hunk 1]
```diff
@@ -25,7 +18,80 @@ function submitForm(event) {
...
+  $row.appendChild($columnOne);
+  $columnOne.appendChild($img);
+  $row.appendChild($columnTwo);
+  $columnTwo.appendChild($title);
+  $columnTwo.appendChild($notes);
+
+  return $entry;
+}
+
+function DOMContentLoaded(event) {
```
Comment (BrettAlbright) [inline js/main.js]: I would five this function a different name, and one with a lowercase first letter.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Looks Great Randy! Just one small update!

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**